be/src/exec/parquet/column_reader.h

```diff
@@ -27,6 +30,269 @@ struct ColumnReaderOptions {
     std::string timezone;
 };
 
+// When doing decimal convert, source scale may not equal with destination scale,
+// when destination scale is greater than source, source unscaled data will be scaled up
+// to match destination scale.
+enum DecimalScaleType {
+    NO_SCALE,
```

Comment:
should start with `k`, such as `kNoScale`

Reply:
done